src/class_sqlThread.py

```diff
@@ -47,7 +47,7 @@ def run(self):  # pylint: disable=too-many-locals, too-many-branches, too-many-s
             self.cur.execute(
                 '''CREATE TABLE subscriptions (label text, address text, enabled bool)''')
             self.cur.execute(
-                '''CREATE TABLE addressbook (label text, address text)''')
+                '''CREATE TABLE addressbook (label text, address text, UNIQUE(address) ON CONFLICT REPLACE)''')
```

Comment:
it needs to be `ON CONFLICT IGNORE`, otherwise the checks for row count won't work correclty.